plugins/org.python.pydev/tests_completions/org/python/pydev/ast/codecompletion/revisited/ModulesManagerTest.java

```diff
@@ -300,6 +298,29 @@ public void testGetAllModulesStartingWithRemoval() {
         assertEquals(0, manager.getAllDirectModulesStartingWith("ab").size());
     }
 
+    public void testBuildKeysForRegularEntries() {
+        ProjectModulesManager modulesManager = (ProjectModulesManager) nature2.getAstManager().getModulesManager();
+        String pythonpath = TestDependent.TEST_PYSRC_TESTING_LOC + "\\testWoInit\\";
+        ProjectStub project = new ProjectStub("testProjectStubRefactoring", pythonpath, new IProject[0],
+                new IProject[0]);
+        IProgressMonitor monitor = getProgressMonitor();
+        modulesManager.pythonPathHelper.setPythonPath(pythonpath);
+        ModulesFoundStructure modulesFound = modulesManager.pythonPathHelper.getModulesFoundStructure(project, monitor);
+        PyPublicTreeMap<ModulesKey, ModulesKey> keys = ModulesManager.buildKeysFromModulesFound(monitor, modulesFound);
+
+        ModulesManager.buildKeysForRegularEntries(monitor, modulesFound, keys, false);
+
+        Set<String> expectedKeyNames = new HashSet<String>();
+        expectedKeyNames.add(".__init__"); // testWoInit.__init__
```

Comment:
This is not correct... We should not have a `.__init__` module (please make sure that you test for that).